util/Pos/Util/Util.hs

```diff
@@ -170,6 +172,11 @@ parsecError = toParsecError . Left
 liftE :: TH.Lift a => Either Text a -> TH.ExpQ
 liftE = TH.lift <=< toTemplateHaskellError
 
+pattern Exc :: Exception e => e -> SomeException
```

Comment:
Adding a Haddock comment and probably a usage example would be very useful, because this feature is not well-known to majority of people.